apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableNotToContainEntriesExpectationsTest.kt

```diff
@@ -31,28 +32,30 @@ class IterableNotToContainEntriesExpectationsSpec :
             else expect.notToContain.entries(a, *aX)
     }
 
-    @Suppress("unused", "UNUSED_VALUE")
-    private fun ambiguityTest() {
-        var list: Expect<List<Number>> = notImplemented()
-        var nList: Expect<Set<Number?>> = notImplemented()
-        var subList: Expect<ArrayList<Number>> = notImplemented()
-        var star: Expect<Collection<*>> = notImplemented()
+}
 
+@Test
+@Suppress("unused", "UNUSED_VALUE")
```

Comment:
Can you please change this to, looks like the Kotlin compiler emits other warnings by now:
```suggestion
@Suppress("AssignedValueIsNeverRead")
```